db/schema.rb

```diff
@@ -21,6 +21,14 @@
     t.datetime "updated_at",  null: false
   end
 
+  create_table "deviceorama_hive_attributes", force: :cascade do |t|
```

Comment:
Does it need to be 'deviceorama'... ? Would this be sufficient as hive_attributes?